libnavigation-core/src/main/java/com/mapbox/navigation/core/navigator/NavigatorMapper.kt

```diff
@@ -252,7 +252,7 @@ private fun MutableList<BannerComponent>.mapToDirectionsApi(): MutableList<Banne
                 .abbreviationPriority(it.abbrPriority)
                 .active(it.active)
                 .directions(it.directions)
-                .imageBaseUrl(it.shield?.baseUrl)
+                .imageBaseUrl(it.imageBaseUrl)
```

Comment:
Is it ok to not use the one coming from the shield? 🤔

Reply:
Yes, the Mapbox Design shield has its own field, this is for the legacy shields system.